past_data_analysis.py

```diff
@@ -0,0 +1,81 @@
+import argparse
+
+from typing import List
+from afdd.utils import * 
+from afdd.models import *
+import psycopg
+import os
+import json
+
+def analyze_past_data(conn: Connection, start_time: str, end_time: str, rule_id: int):
+  """
+  Creates a json file with all anomalies that happened between start and end time for a specified rule.
+  """
+  graph = load_graph(devices="kaiterra_example.ttl")

```

Comment:
Lets pass in the path to the graph file as an argument, read the graph in main, then pass it to this function